docs/docs/dev/policies/timeout.md

```diff
@@ -88,3 +126,89 @@ Request timeouts are configured on the Envoy routes and may select a different T
 
 Mesh configures an idle timeout on the HTTPConnectionManager, but doesn’t consistently use the Timeout policy values for this, so the semantica are ambiguous.
 There’s no policy that configures the idle timeout for downstream connections to the Gateway.
+
+## Inbound timeouts
+
+Current policy configures timeouts only on the outbound side.
+Timeouts on the inbound side have constant values:
+
+```yaml
+connectTimeout: 10s 
+tcp:
+  idleTimeout: 2h
+http:
+  requestTimeout: 0s
+  idleTimeout: 2h
+  streamIdleTimeout: 1h
+  maxStreamDuration: 0s
+```
+
+These timeouts are either disabled or 2 times bigger than outbound default values
+to not interfere with the user's smaller values.
```

Comment:
I'm not sure that's useful information you might just want to say it differently when you offer users to modify it

Reply:
I think it makes sense to simply delete these lines, conf just above is self-explanatory